src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -16,6 +17,14 @@
           Condition="'$(RestorePackages)'=='true' AND '$(IsTestProject)' == 'true' AND '$(TestRuntimeProjectJson)' != ''">
     <Exec Command="$(DnuRestoreCommand) &quot;$(TestRuntimeProjectJson)&quot;" StandardOutputImportance="Low" CustomErrorRegularExpression="^Unable to locate .*" />
   </Target>
+  
+  <Target Name="ValidateTestRuntimeDependencies"
+          AfterTargets="RestoreTestRuntimePackage"
```

Comment:
I suppose it could be run up front at the same time we're restoring when building from the root.  Similar to how we disable per-project restore when building from the root.